### DISCUSSION TITLE
Libfmt download error leads to Cmake configure failure

### DISCUSSION THREAD

**Discussion by 0penBrain** [FreeCAD/FreeCAD#9571] [Code contribution discussions]
Interesting case in this job : https://github.com/FreeCAD/FreeCAD/actions/runs/4948478344/jobs/8849254906
I don't understand everything coming into play there, but it looks like Cmake tried to download some part of libfmt and fails as a server error occurred.
Just wondering if this case could be better handled (though it seems pretty rare, only case I ever saw in CI).

@chennes pinging you as I found you authored the Cmake lines. :smile:

**Comment by chennes** [comment]
If we add libfmt as a dependency and install it before the cMake run we will avoid this.

**Reply by 0penBrain** [reply]
Ok thanks. Is there one setup more interesting to run from a CI point of view compared to the other? Or is it equal?
